docs/index.md

```diff
@@ -2,16 +2,27 @@
 
 Pulse displays metrics that provide insights about the current and historic performance of your software delivery process. To calculate these metrics, Pulse must collect information from key events of your particular software development workflow.
 
-Currently, Pulse supports a push-based integration with your workflow and the recommended way of sending the necessary data to Pulse is by using the Pulse CLI.
+Currently, Pulse supports a push-based integration with your workflow and the recommended way of reporting the necessary events to Pulse is by using the Pulse CLI.
 
-However, in some scenarios it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are sending data from providers that only support webhooks. For these situations, you can call an HTTP POST webhook instead. For example:
+## Before you begin
 
-`https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>`
+Consider the following before integrating your workflow with Pulse:
 
-!!! important
-    **Please let us know if you are planning on pushing events to Pulse using the webhook** so we can give you more detailed instructions on how to use it.
+-   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are reporting events from providers that only support webhooks. For these situations, you can call an HTTP POST webhook instead. For example:
+
+    ```test
+    https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>
+    ```
+
+    !!! important
+        **Please let us know if you are planning on reporting events to Pulse using the webhook** so we can give you more detailed instructions on how to use it.
+
+-   When reporting events to Pulse you can associate each event with a "system". Depending on your use case, the system can be a repository, an application or service, a product, or any other entity or group of entities in your organization.
+
+    !!! important
+        Pulse does not use this information yet, but as we develop new features you will be able to use this information to view metrics for different scopes. This means that the way you group events in systems **determines the most granular scope** that you will be able to apply to your metrics on Pulse.
```

Comment:
I think we should say this should be the smallest unit people want to possibly see data aggregated about and that ids need to be unique inside this unit.

Reply:
Yes, it makes sense. It's better to collect data with the highest granularity to make the grouping of the information from several "systems" on the front end as flexible as possible and not limit any use case.

Taken together with [Rafael's comment](https://github.com/codacy/pulse-user-docs/pull/1#issuecomment-733556970), this means that there isn't much to be decided by the customer - which is good!